odoo/addons/base/ir/ir_model_view.xml

```diff
@@ -225,6 +225,7 @@
                                 <field name="name"/>
                                 <field name="field_description"/>
                                 <field name="model_id" attrs="{'readonly': [('state','!=', 'manual')]}"/>
+                                <field name="model" invisible="1"/>
```

Comment:
This is empty in creation even after choosing a model (`model_id`), this is a problem at python level. I do not know if there is any other place where I can use this `widget="field"`.